zebra-consensus/src/transaction.rs

```diff
@@ -6,13 +6,16 @@ use std::{
     task::{Context, Poll},
 };
 
+#[allow(unused_imports)]
```

Comment:
I'd strongly prefer not to add all of these annotations to the source code.  It just creates more work for us later.